src/VisualStudio/TestUtilities2/project.json

```diff
@@ -24,7 +24,8 @@
     "Microsoft.VisualStudio.Text.Logic": "15.0.26014-alpha",
     "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
     "Microsoft.VisualStudio.Text.UI": "15.0.26014-alpha",
-    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.26014-alpha"
+    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.26014-alpha",
+    "System.Composition": "1.0.31"
```

Comment:
I don't understand why this had to be added so many places - why didn't it flow transitively from `src/Workspaces/Core/Portable/project.json`?